src/pages/resultsView/survival/SurvivalChart.tsx

```diff
@@ -320,6 +329,19 @@ export default class SurvivalChartExtended
         );
     }
 
+    @observable visible = false;
+
+    // determine whether visibility is controlled
+    // by a store component or by local state
+    @computed get visibilityState() {
+        return this.visible;
+    }
+    @action setVisibilityState(visible: boolean | undefined) {
+        if (visible !== undefined) {
+            this.visible = !!visible;
+        }
+    }
```

Comment:
I don't see a reference to any of these variables (`visible`, `visibilityState`, `setVisibilityState`). Do we actually use them somewhere?

Reply:
Removed this part since it became obsolete.